### PR TITLE
UI

### PR DESCRIPTION
**Pull request by WebGenius912**

### CODE REVIEW

File: ui/src/components/Home.tsx [hunk 1]
```diff
@@ -86,22 +91,27 @@ const Home: React.FC = () => {
             </span>
             {wallet.connected && <WalletDisconnectButton />}
           </div>
-        </Grid>
-        <Grid item xs={12} className="bg-[#334155] p-10">
-          <TrackUpload
+        </Grid> */}
+        <Grid item xs={10} className="bg-[#334155] p-10">
+          {/* <Routes> */}
```
Comment (AtlasPilotPuppy) [inline ui/src/components/Home.tsx:96]: clean commented code

File: ui/src/components/Home.tsx [hunk 2]
```diff
@@ -86,22 +91,27 @@ const Home: React.FC = () => {
...
+          {/* <Route path="/upload" element={<Wallet />}/> */}
+            {/* <Route path="/upload" element={<TrackUpload ipfs={ipfsClient} connection={connection} wallet={wallet} program={program}/>}/> */}
+            {/* <Route path="/allimage" element={<TracksView userKey={wallet.publicKey} connection={connection} client={ipfsClient} program={program}/>}/> */}
```
Comment (AtlasPilotPuppy) [inline ui/src/components/Home.tsx:99]: clean commented code

File: ui/src/components/TrackUpload.tsx [hunk 3]
```diff
@@ -1,14 +1,88 @@
...
-  );
-};
+    );
+
+  //   if(anchorWallet) {
+  //     console.log('anchorWallet-UploadTrack=>', anchorWallet)
+  //     return (
+  //         <div>here is all tracks</div>
+  //         // <TracksView
+  //         //     connection={connection}
```
Comment (AtlasPilotPuppy) [inline ui/src/components/TrackUpload.tsx:72]: clean commented code

**Review by AtlasPilotPuppy — CHANGES_REQUESTED** [review]

File: ui/src/components/Home.tsx (hunk 1 above)
Comment (WebGenius912) [reply ui/src/components/Home.tsx:96]: We don't use Home.tsx anymore.  I have defined all everything in App.tsx. so we have to remove Home.tsx after

File: ui/src/components/Home.tsx (hunk 2 above)
Comment (WebGenius912) [reply ui/src/components/Home.tsx:99]: This is same

File: ui/src/components/TrackUpload.tsx (hunk 3 above)
Comment (WebGenius912) [reply ui/src/components/TrackUpload.tsx:72]: that is not related to it. It needs for testing
So I will update in next step

**Comment by AtlasPilotPuppy** [comment]
open

**Pull request merged**